src/OtherQuantityHandler.cpp

```diff
@@ -273,6 +276,20 @@ void OtherQuantityHandler::DefineQuantities() {
             real_t *ncold = this->unknowns->GetUnknownData(this->id_ncold);
             this->tracked_terms->T_cold_radiation->SetVectorElements(qd->StoreEmpty(), ncold);
         );
+    // Magnetic ripple resonant momentum
+    if (tracked_terms->f_hot_ripple_Dxx != nullptr) {
+        len_t nModes = tracked_terms->f_hot_ripple_Dxx->GetNumberOfModes();
+        DEF_FL("fluid/f_hot_ripple_m", "Magnetic ripple poloidal mode number for f_hot", qd->Store(this->tracked_terms->f_hot_ripple_Dxx->GetPoloidalModeNumbers()););
```

Comment:
if we move the toroidal field quantities to radialgrid, we could rename these to just fluid/ripple_m etc.

You would probably still want to check whether f_hot_ripple_Dxx _or_ f_re_ripple_Dxx are not nullptr, since in principle we may want to support running f_re without f_hot.